data/qinxin

```diff
@@ -0,0 +1 @@
+qinxin.com
```

Comment:
The domain `qinxin.com` can not be resolved.

Reply:
www.qinxin.com

Reply:
> [www.qinxin.com](http://www.qinxin.com)

Still can not resolve

Reply:
Sorry I input incorrect domain.
www.qixin.com